confdb/aclocal_libfabric.m4

```diff
@@ -0,0 +1,26 @@
+dnl
+dnl/*D 
+dnl PAC_LIB_FABRIC - Check for Libfabric dependence library
```

Comment:
This looks good to me, but could you make it more generic? Say `PAC_LIB_DEPS([foo])` so we can use it for more than just libfabric?